csrc/scheduler/cutlass.cpp

```diff
@@ -67,17 +71,26 @@ bool CutlassScheduler::canScheduleCompileTime(Fusion* fusion) {
     return false;
   }
 
-  // Check if fusion has a supported matmul pattern
-  if (!hasSupportedMatmulPattern(fusion)) {
+  const cudaDeviceProp* device_prop = at::cuda::getCurrentDeviceProperties();
+  if (device_prop->major < 10) {
```

Comment:
GB200 and GB300 are sm 10.0 and sm 10.3 respectively. sm 11 and sm 12 are not compatible.

```suggestion
  if (device_prop->major != 10 || !(device_prop->minor == 0 ||  device_prop->minor == 3)) {
```